test/functional/healthcheck.test.ts

```diff
@@ -46,27 +38,30 @@ describe('proxy requests originating from behind the broker client', () => {
     });
   });
 
-  beforeEach(() => {
-    jest.resetModules();
-  });
-
   afterAll(async () => {
-    await utilServer.httpServer.close();
-    await client.close();
+    await tws.server.close();
     setTimeout(async () => {
-      await server.close();
+      await bc.client.close();
     }, 100);
+    await new Promise<void>((resolve) => {
+      bc.client.io.on('close', () => {
+        resolve();
+      });
+    });
 
+    setTimeout(async () => {
+      await bs.server.close();
+    }, 100);
     await new Promise<void>((resolve) => {
-      server.io.on('close', () => {
+      bs.server.io.on('close', () => {
         resolve();
       });
     });
   });
 
-  it('server healthcheck', async () => {
+  it.skip('server healthcheck', async () => {
     const response = await axios.get(
```

Comment:
Why skip?

Reply:
Looks like this file is not ready, maybe take it out of this PR?

Reply:
> Why skip?

For some reason the test stop working locally (infinite retry of the broker client).
It makes sense to take it out of the PR and commit when it works. Done with 2aedffc.